isis/src/control/objs/BundleAdjust/BundleAdjust.cpp

```diff
@@ -1857,6 +1857,8 @@ namespace Isis {
                                      BundleMeasure &measure,
                                      BundleControlPoint &point) {
 
+    std::cout << "Computing partials for [" << measure.cubeSerialNumber() << "] in [" << point.id() << "]." << std::endl;
```

Comment:
I think it probably makes sense to leave this debugging print statement in here, just want to note that it is in here.

Reply:
Oops I thought i took all of these out.